### PR TITLE
Ramonawalls purlpolicychanges

### PR DESCRIPTION
**Pull request by ramonawalls**
Members of the technical WG, please review this pull request. I updated the PURL policy page and completely revamped the page called BOPURLDomain.html, which was pretty useless before.

This PR should resolve issues #620 and #414.

### CODE REVIEW

**Comment by jamesaoverton** [comment]
Thanks Ramona for writing this up. Sorry that it has taken me so long to take a proper look at it.

I think it's substantially correct, but I'm worried that the terminology is inconsistent. "PURL" and "prefix" are both being used in these docs to mean "IDSPACE" (itself not a great term), where "PURL" usually means a special kind of IRI.

For instance, in one of these documents "process for requesting a new PURL" means the process for requesting a new IDSPACE for a new ontology (a heavyweight process, requiring OFOC approval), not the process of assigning a PURL to a new release (for example, which is lightweight, and requires no OFOC approval). The previous sentence talks about the PURL for a class (light). The rest of the document is about requesting an IDSPACE (heavy).

Can I dive in an make substantial revisions to harmonize the terminology here?

File: docs/OBOPURLDomain.md [hunk 1]
```diff
@@ -1,11 +1,39 @@
 ---
 layout: doc
 id: OBOPURLDomain
-title: OBOPURLDomain
+title: OBO_PURL_Creation
```
Comment (cmungall) [inline docs/OBOPURLDomain.md]: We can make this human-friendly and use spaces

(we should do this for all docs, they are camel case as the title was auto-generated from the filename originally)

File: docs/OBOPURLDomain.md [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
-The [OBO ID Policy](https://obofoundry.org/id_policy.shtml) describes the general schema for which purls are then create, and what the format of individual purls are.
+Every ontology in OBO has a PURL, each version has a PURL, and every class also gets a PURL. The PURLs for ontologies can be found on individual ontology home pages.
+
+The [OBO ID Policy](http://www.obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) describes the requirements and process for requesting a new PURL.
+
+For technical details of how the OBO PURL system works, and the format of individual purls, refer to https://github.com/OBOFoundry/purl.obolibrary.org/.
+
+# SOP for new PURL creation #
+
+This SOP is intended for use by the OBO Foundry Technical Working Group. If you are an ontology developer wanting to request a new PURL, please see the [PURL Request Guidelines](http://www.obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.htm). Savvy users may follow some of these steps to create their own ontology metadata files and then make a pull request, following the two week period of public comment.
```
Comment (cmungall) [inline docs/OBOPURLDomain.md]: .htm links don't work. Use .html, or (better) omit altogether

File: docs/OBOPURLDomain.md [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+
+## Prerequisites ##
+1. Someone has requested a new PURL, following the [PURL Request Guidelines](http://www.obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.htm), i.e. they have already sent an email to [obo-discuss](mailto:obo-discuss@googlegroups.com) and created an issue on the [tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues).
+
+## PURL request review process ##
+1. All members of the OBO community should respond to requests on obo-discuss as appropriate. This is particurly important if the new resource overlaps with an ontology you work on.
```
Comment (cmungall) [inline docs/OBOPURLDomain.md]: typo

File: docs/Policy_for_OBO_namespace_and_associated_PURL_requests.md [hunk 4]
```diff
@@ -18,38 +23,36 @@ The technical working group reserves the right to make decisions based on their
...
-  1. It is expected that solicitation of a prefix is done _before_ the prefix is used for identifiers. A common strategy is to develop an ontology, request a prefix, and translate the initial URIs used to the PURLs some time before the initial release. **There is no guarantee that you will be granted your prefix, even if you have been using it in your file**.
+  1. It is expected that solicitation of a prefix is done _before_ the prefix is used for identifiers. A common strategy is to develop an ontology, request a prefix, and translate the initial URIs used to the PURLs some time before the initial release.  **There is no guarantee that you will be granted your prefix, even if you have been using it in your file**.
 
-## Information needed for a prefix request ##
 
-You must first make your ontology available via a stable URL. Provide this to us, together with the following metadata:
+## Requesting the prefix ##
+
+  1. First send an email to [obo-discuss](mailto:obo-discuss@googlegroups.com) with your request to allow community feedback (you may need to [apply to join group](https://groups.google.com/forum/#!forum/obo-discuss) first.)
```
Comment (cmungall) [inline docs/Policy_for_OBO_namespace_and_associated_PURL_requests.md]: Why not have them first submit the issue, then the issue can be referenced in the email?

**Comment by ramonawalls** [comment]
James, you are totally right. Thanks for taking a rigorous look at the wording. Please feel free to dive in and make whatever edits you think are appropriate.

Now Chris has (I think) made some changes, and there are these things called "coversations" that I did not even know existed in github. @jamesaoverton perhaps you can resolve those as you make edits? I may not get around to figuring out how for a few days.

File: docs/OBOPURLDomain.md (hunk 2 above)
Comment (ramonawalls) [reply docs/OBOPURLDomain.md]: agreed

File: docs/OBOPURLDomain.md (hunk 3 above)
Comment (ramonawalls) [reply docs/OBOPURLDomain.md]: I can't see this one

File: docs/Policy_for_OBO_namespace_and_associated_PURL_requests.md (hunk 4 above)
Comment (ramonawalls) [reply docs/Policy_for_OBO_namespace_and_associated_PURL_requests.md]: Great idea!

Comment (ramonawalls) [reply docs/Policy_for_OBO_namespace_and_associated_PURL_requests.md]: The policy predates our use of github. Time to bring it up to date.

**Comment by jamesaoverton** [comment]
Yes, I'll take it from here. Thanks @ramonawalls!

**Comment by jamesaoverton** [comment]
I tried to really simplify this text, and harmonize the terminology. It probably still needs another pass.

File: docs/OBOPURLDomain.md (hunk 3 above)
Comment (cmungall) [reply docs/OBOPURLDomain.md]: There should be no closing `##`s

Comment (cmungall) [reply docs/OBOPURLDomain.md]: also good practice to leave newlines either side

**Comment by jamesaoverton** [comment]
Thanks again @ramonawalls. What you wrote was very helpful. Team effort!

**Comment by yongqunh** [comment]
Agree. Thanks.

**Pull request merged**